src/test/ClusterDeployer/cluster.sh

```diff
@@ -26,7 +26,7 @@ BuildImage()
   fi
 
   pushd "../../../out/build.prod"
-  sudo docker build -f dockers/cluster-deployer/Dockerfile -t sfosscontainerreg.azurecr.io/clusterdeployer . 
+  sudo docker build -f dockers/cluster-deployer/Dockerfile -t microsoft/service-fabric-run-ubuntu/clusterdeployer . 
```

Comment:
This isn't really a proper name.  "microsoft/image-name" usually implies that there is an image hosted on Docker Hub at that location.  Having two forward slashes "microsoft/image-name/foo" doesn't map to anything in Docker Hub.  If we want to host the container, it should be named "microsoft/service-fabric-run-ubuntu" (no "/clusterdeployer"), and we need to request a Docker Hub repo creation.  Otherwise it should just be tagged "service-fabric-run-ubuntu" locally.

Reply:
sure